packages/core/contracts/financial-templates/optimistic-distributor/OptimisticDistributor.sol

```diff
@@ -361,22 +360,6 @@ contract OptimisticDistributor is Lockable, MultiCaller, Testable {
      *          MAINTENANCE FUNCTIONS           *
      ********************************************/
 
-    /**
-     * @notice Sets address of MerkleDistributor contract that will be used for rewards distribution.
-     * MerkleDistributor address can only be set once.
-     * @dev It is expected that the deployer first deploys MekleDistributor contract and transfers its ownership to
-     * the OptimisticDistributor contract and then calls `setMerkleDistributor` on the OptimisticDistributor pointing
-     * on now owned MekleDistributor contract.
-     * @param _merkleDistributor Address of the owned MerkleDistributor contract.
-     */
-    function setMerkleDistributor(MerkleDistributor _merkleDistributor) external nonReentrant() {
```

Comment:
interesting. so removing this means each OD has exactly 1 Merkile distributor and it is the owner there. pretty clean solution.